framework/wazuh/core/cluster/common.py

```diff
@@ -69,6 +69,7 @@ def __init__(self, total=0):
         self.total = total  # total of bytes to receive
         self.received = 0  # number of received bytes
         self.cmd = ''  # request's command in header
+        self.flag_divided = b''  # request's command flag to indicate a msg division
```

Comment:
```suggestion
        self.flag_divided = b''  # request's command flag to indicate a msg division
        self.divide_flag = b' d'
        self.end_divide_flag = b' e'
```

Reply:
Class variables instead of instance variables